lib/index.d.ts

```diff
@@ -380,7 +380,7 @@ declare namespace script {
          * 
          * @returns a wrapped function.
          */
-        mustCall(func: Function, count: number): Function;
+        mustCall<T extends (...args: any[]) => any>(func: T, count: number): T;
```

Comment:
This *might* impose some version constraints on the typescript version that can consume the typing info.

Reply:
We only support latest TS versions. Is this a breaking change or not because it wasn't working to begin with?

Reply:
Hi, sorry I didn't get back earlier. I agree with your assessment; this should be treated as a bugfix IMHO because it produces a more correct typing result.